tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -2,7 +2,7 @@ common:
   depends_on: spi
   tags: drivers spi
   platform_exclude: seeeduino_xiao serpente arduino_nano_33_iot atsamr21_xpro
-    atsamd21_xpro arduino_zero adafruit_trinket_m0
+    atsamd21_xpro arduino_zero adafruit_trinket_m0 arduino_mkrzero
```

Comment:
I don't remember this test requiring DMA support. Are you sure about this?

Reply:
DMA is selected by Kconfig if using SAM0's UART/SPI drivers with enabling ASYNC option.
That's right as you say, "Tests is not requiring DMA support.".
 I fixed the commit message because it is not a good description.

https://github.com/zephyrproject-rtos/zephyr/blob/f7171e20b33ee91ac9c5efc30d3a491f05ce177e/drivers/spi/Kconfig.sam0#L10

https://github.com/zephyrproject-rtos/zephyr/blob/f7171e20b33ee91ac9c5efc30d3a491f05ce177e/drivers/serial/Kconfig.sam0#L17

In commit ea36f9b and f4c5bdf,  SAMD21 boards were excluded from spi_loopback test and uart_async_api test.
I followed that policy in this PR.

Reply:
Thanks for clarifying

Reply:
Thanks for clarifying